test/models/diffusion/test_song_unet.py

```diff
@@ -133,6 +144,41 @@ def test_song_unet_constructor(device):
     except ValueError:
         pass
 
+    try:
```

Comment:
This try-except formulation is tricky to follow. Better to use https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.raises

```
with pytest.raises(Exception):
    model = ...
```